### PR TITLE
feat: :wrench: Add support for NodeMCU amd uPesy

### PR DESCRIPTION
**Pull request by Wilaz**

### CODE REVIEW

File: include/Chipset.h [hunk 1]
```diff
@@ -207,6 +207,16 @@
 #ifdef OPENSHOCK_FW_BOARD_DFROBOTFIREBEETLE2ESP32E
 #define OPENSHOCK_BYPASSED_GPIO(pin) ((pin) == 2 || (pin) == 5)
 #endif
+#ifdef OPENSHOCK_FW_BOARD_NODEMCU32S
+#define OPENSHOCK_BYPASSED_GPIO(pin) ((pin) == 15 || (pin) == 2)
```
Comment (hhvrc) [inline include/Chipset.h:211]: Why are these pins bypassed?

Comment (Wilaz) [reply include/Chipset.h:211]: Pin 2 is for the built in LED and pin 15 was just because the other boards do it. You can remove pin 15 if you'd like :3

Comment (Wilaz) [reply include/Chipset.h]: It should be good now!

**Comment by Wilaz** [comment]
@hhvrc Can this be merged yet?

**Pull request closed without merging**